### ISSUE TITLE
Playground: Uses (very old) express-serve-static-core/index.d.ts instead of @types/express-serve-static-core

### ISSUE THREAD

**Issue by Gaelan** [microsoft/TypeScript-Website#2119]
As a workaround for a circular dependency, most of the typings for Express are actually in `@types/express-serve-static-core`, which `@types/express` depends on. Note that there's not really a `express-serve-static-core` package at runtime—it's purely invented by DefinitelyTyped to make typings work.

However, there *is* an `express-serve-static-core` package on NPM, with an `index.d.ts` that's a 5-year-old copy of the DefinitelyTyped typings. For some reason, the playground is picking up that package, instead of `@types/express-serve-static-core`, so it's missing features that have been added to the typings since then—for example, the template-string-literal-based parsing of route parameters.

**Repro:** https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbwEoQK4wKZQL5wGZQQhwDkGAHmFBgM40kDcAUEwMYQB2N8h6WcAXjgo+UABQBKZk16YoAOgDmGGGJIB6AFzAAJiQA0cMdQCOh6jQmCAfIiZwHcdergABGDQC0FMBlYxvKEIoe0dTeTAAQyhIkBp5EGA6YA5FADFgDAAbHTgmbAkgA

In the playground, write TypeScript code that imports `express`.

**Expected behavior:**

The playground uses the current typings from `@types/express-serve-static-core`—the ones that get installed when you run `npm i -D @types/express`.

**Actual behavior:**

The playground uses the ancient typings from `express-serve-static-core`.

**Comment by orta** [comment]
Aye, makes sense. 

I think we should blocklist grabbing `express-serve-static-core` in the ata package and just go straight to the `@types`

**Comment by seansfkelley** [comment]
I just ran into this issue as well. Very weird and frustrating. I filed https://github.com/microsoft/TypeScript-Website/issues/2328 as an improvement that would have helped me figure this out faster.

Unfortunately, I think this means the Playground can't be used to file bugs on these type definitions, since they won't necessarily reproduce the same way, if at all (which is what I'm seeing for my typing issue).
